cmd/lambda/leases/validate.go

```diff
@@ -78,7 +78,10 @@ func validateLeaseFromRequest(context *leaseValidationContext, req *http.Request
 	}
 
 	if spent > context.principalBudgetAmount {
-		validationErrStr := fmt.Sprintf("Unable to create lease: User principal %s has already spent %f of their principal budget", requestBody.PrincipalID, math.Round(context.principalBudgetAmount))
+		validationErrStr := fmt.Sprintf(
```

Comment:
The test for this error messages was failing. Figured I'll clean up the message while I'm in here
(rounds to 2 decimal places, includes actual spend)